### ISSUE TITLE
Add ability to set scroll parent

### ISSUE THREAD

**Issue by SvyatovetsD** [adazzle/react-data-grid#1432]
- Have you already searched for similar issues? Please help us out and double-check first!

- Also, please don't be that person who deletes this template. It's here for a reason.

- Thanks!

---

### Which version of React JS are you using?

✅ Officially supported ✅
- [ ] v15.4.x

⚠️ Not officially supported, expect warnings ⚠️
- [ ] v15.5.x
- [ ] v15.6.x

☣️ Not officially supported, expect warnings and errors ☣️
- [x] v16.x.x

---

### Which browser are you using?

✅ Officially supported ✅
- [ ] IE 9 / IE 10 / IE 11
- [ ] Edge
- [x] Chrome

⚠️ Not officially supported, but "should work" ⚠️
- [ ] Firefox
- [ ] Safari

---

### I'm submitting a ...

- [ ] 🐛 Bug Report
- [x] 💡 Feature Request

> 👋 Need general support? Not sure about how to use React itself, or how to get started with the Grid?
> Please do not submit support request here. Instead see

> https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md

---

### Issue Details
By default editor portal injects editor to document.body. It leads to editor positioning problems if grid scroll parent is another element.

```
Please include:
- What the current behavior is
Portal always injects editor to body
- What the desired behvaior is
Have ability to override default scroll parent
- (If Bug) Steps to reproduce the issue
- (If Feature) The motivation / use case for the feature

We especially love screenshots / videos of problems, and remember
The Best Issue Is A Pull Request™
```

**Comment by amanmahajan7** [comment]
A new `editorPortalTarget` prop has been added to handle this case.  We also released an alpha version 6.0.2-0. Please give a shot and lets us know if everything is working as expected. Thanks

**Comment by SvyatovetsD** [comment]
@amanmahajan7 thanks, works great
